.github/workflows/ci.yml

```diff
@@ -39,6 +43,11 @@ jobs:
         name: Test
         with:
           command: test
```

Comment:
I only noticed this now but testing implies a build so we could fold this two

Reply:
One build is debug while the other is release. I reordered them so that tests come quicker.